### PR TITLE
add playitgg

### PR DESCRIPTION
**Pull request by Dysta**
https://playit.gg is a tunnelling service for self hosted minecraft server. It allow you to publish it to the net without opening your ports and IP

### CODE REVIEW

**Comment by dragonfire1119** [comment]
Thank you for the pull request. Can you add this app to the README?

**Comment by Dysta** [comment]
✅

File: Apps/playitgg/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    volumes:
+    - type: bind
+      source: /DATA/AppData/$AppID/secrets
+      target: /secrets
+      bind:
+        create_host_path: true
+x-casaos:
+  architectures:
+    - amd64
+    - arm
```
[the remark this answers is not in the document]
Comment (Dysta) [reply Apps/playitgg/docker-compose.yaml:16]: bad copy/past, i'll fix this

File: Apps/playitgg/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: playitgg
```
Comment (dragonfire1119) [inline Apps/playitgg/docker-compose.yaml:1]: This should be big-bear-playitgg. So it doesn't interfere with another app store.

**Comment by dragonfire1119** [comment]
Also, I noticed it's not a Docker image. Do you plan on keeping it maintained?

**Comment by dragonfire1119** [comment]
Also, I noticed it's your Docker image. Do you plan on keeping it maintained?

**Comment by dragonfire1119** [comment]
Thank you! Please don't hesitate to contribute further, as it only enhances the community. :)

Published at: https://community.bigbeartechworld.com/t/added-playitgg-to-bigbearcasaos/283

**Pull request merged**